go/oasis-node/cmd/node/node.go

```diff
@@ -398,21 +391,6 @@ func (n *Node) startRuntimeWorkers() error {
 		return err
 	}
 
-	// Start the public consensus services worker.
```

Comment:
External gRPC server, TLS address in registry and similar are not needed anymore after this change. Due to the amount of changes, it will be removed in a followup PR.